File: php/post.php
<?php 
$endpoint = "http://example.com";

$data = Array(
    "retain"=> 1,
	"counters" => Array(
		"PBX/line status.0" => 1,
		"PBX/line status.1" => 0,
		"PBX/version" => "Mock Phone System 1.0"
	),
    "apikey" => "MDAwMDAwMDAxM0FDRDhDQj=="

);
$curl = curl_init($endpoint."/restj/ncrest/openmon/update"); 

$options = array(
	CURLOPT_POST => true, 
	CURLOPT_HTTPHEADER => array('Content-type: application/json; charset=utf-8'),
	CURLOPT_POSTFIELDS => json_encode($data),
	CURLOPT_RETURNTRANSFER => true
);
curl_setopt_array( $curl, $options );

$result = curl_exec($curl); 

if(!curl_errno($curl)){ 
  echo 'Response: '.$result; 
} else { 
  echo 'Curl error: ' . curl_error($curl); 
} 
curl_close($curl); 
?>

File: python/post.py
import urllib.request
from urllib.error import HTTPError, URLError
import json

ENDPOINT = "http://example.com"

data = {
    "retain": 1,
    "counters": {
        "crm/day-orders": 121,
        "crm/today-emails": 234
    },
    "apikey": "MDAwMDAwMDAxM0FDRDhDQj=="
}

try:
    url = ENDPOINT + "/restj/ncrest/openmon/update"
    opener = urllib.request.build_opener()
    opener.addHeaders = [('Content-Type', 'application/json; charset=utf-8')]
    u = opener.open(url, json.dumps(data).encode('utf-8'))
    print("Response: " + u.read().decode('utf-8'))
except HTTPError as e:
    print("HTTPError: %s" % e.code)
except URLError as e:
    print(e)
